ci/azure-pipelines-template.yml

```diff
@@ -47,14 +47,37 @@ jobs:
     displayName: Clippy
     condition: ne( variables['rustup_toolchain'], 'nightly' )
 
-  - ${{ if ne(parameters.name, 'Windows') }}:
+  - ${{ if eq(parameters.name, 'Linux') }}:
+    - script: |
+        cargo run --bin pyoxidizer -- init --pip-install cryptography ~/pyapp
+        cat ci/pyapp.py | cargo run --bin pyoxidizer -- run ~/pyapp
```

Comment:
https://github.com/indygreg/PyOxidizer/issues/208 is needed for all of these enhancements to the sample app CI.  This approach is taken because otherwise the lines are very long, and also to avoid horrid Windows CMD quoting problems.

Reply:
Fix included in https://github.com/indygreg/PyOxidizer/pull/200